examples/flutter_gallery/lib/welcome/constants.dart

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class Constants {
```

Comment:
I think you're not using these are you? 

You can just have them in the file that uses them. Also camelCase for variable names.

Reply:
Removed